packages/e2e-playwright/models/card.ts

```diff
@@ -90,13 +120,34 @@ class Card {
         await this.cardNumberInput.clear();
     }
 
+    async deleteExpiryDate() {
+        await this.expiryDateInput.clear();
+    }
+
+    async deleteCvc() {
+        await this.cvcInput.clear();
+    }
+
     async typeExpiryDate(expiryDate: string) {
         await this.expiryDateInput.type(expiryDate, { delay: USER_TYPE_DELAY });
     }
 
     async typeCvc(cvc: string) {
         await this.cvcInput.type(cvc, { delay: USER_TYPE_DELAY });
     }
+
+    async pressKeyboardToNextItem() {
+        await this.page.keyboard.press('ArrowDown', { delay: KEYBOARD_DELAY });
+    }
+
+    async pressKeyboardToSelectItem() {
+        await this.page.keyboard.press('Enter', { delay: KEYBOARD_DELAY });
+    }
```

Comment:
Minor thing: These functions look more like a `utils` rather than  specific functions related the `card` model. 
I can foresee we replicating those in different parts of the tests . An idea would be to move them to something like `utils/keyboard.ts`, and reuse them across different tests.

Reply:
Fair point. Will make a utils for this